From: Xiaoxue <xiaoxue@users.noreply.github.com>
Date: Tue, 19 May 2026 09:30:00 +0800
Subject: [PATCH] rockchip: add ECB-PR51 dtb to DTS Makefile

Signed-off-by: Xiaoxue <xiaoxue@users.noreply.github.com>

--- a/target/linux/rockchip/dts/rk356x/Makefile
+++ b/target/linux/rockchip/dts/rk356x/Makefile
@@ -1,6 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0-only
 
 dtb-y += \
+	rk3568-ecb-pr51.dtb \
 	rk3566-radxa-cm3-io.dtb \
 	rk3566-radxa-cm3-rpi-cm4-io.dtb \
 	rk3566-radxa-zero-3.dtb \
